### DISCUSSION TITLE
Errors when testing Pinia store

### DISCUSSION THREAD

**Discussion by drseu55** [vitest-dev/vitest#1539] [Q&A]
### Setup

Vue 3
Vue router 4
Pinia 2

### Description

Hi. I have two Pinia stores. Both of the stores import functions from each other. One of the stores uses axios. I am importing only one of the Pinia stores like the example shown here: [Pinia docs](https://pinia.vuejs.org/cookbook/testing.html). When I start the test with a simple check like `expect(1).toBe(1)`, I am getting this error:

![vitest-error1](https://user-images.githubusercontent.com/19574305/175551251-176893b2-bdd8-474d-b367-5c4f31b2d539.png)

When I comment out the axios part, I am getting this error:

![vitest-error2](https://user-images.githubusercontent.com/19574305/175551701-a9876216-2702-4c9a-90e4-e7bc21d4130e.png)

**Comment by sheremet-va** [comment]
Do you have jsdom as your environment?

**Reply by drseu55** [reply]
Yes, I do have jsdom. I've removed:

```js
define: {
    "window.global": {},
},
```

from vite.config.js and the errors disappeared.
